src/storm/values.yaml

```diff
@@ -36,9 +39,21 @@ supervisor:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  # Define the number of slots/workers per supervisor, each worker/slot is assigned a port
+  slots: 4
   worker:
     heap_memory_mb: 768
     childopts: "-Xmx%HEAP-MEM%m -XX:+PrintGCDetails -Xloggc:artifacts/gc.log -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=1M -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=artifacts/heapdump"
+  memory:
+    capacity: 4096.0 # mb
+  cpu: 
+    #By convention 1 cpu core should be about 100, but this can be adjusted if needed
+    # using 100 makes it simple to set the desired value to the capacity measurement
+    # for single threaded bolts
+    capacity: 400.0
+  security:
+    userid: 1000
```

Comment:
this is a user ide form docker image that we are using can be shared with ui, nimbus and supervisor